components/ContactCard.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div v-if="show">
+    <div class="float-right pointer" @click="copy(contactObj)">
+      Copy
+    </div>
+    <div class="container p-4">
+      <div class="row">
+        <div class="col-2">
+          <BImg
+            src="https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50?s=80"
+            :rounded="false"
+          />
+        </div>
+        <div class="col-10">
+          <div class="ml-3">
+            <div class="h3 pointer" @click="showContact(contactObj)">
+              {{ contactObj.title }} {{ contactObj.firstName }} {{ contactObj.lastName }}
+            </div>
+            <div class="h4">
+              {{ contactObj.designation }}
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row mt-2">
+        <div v-if="contactObj.organization" class="col-12">
+          <span class="h4 pointer" @click="showOrganization(contactObj)">
+            {{ contactObj.organization.name }}
+            {{ contactObj.organization.acronym && `(${contactObj.organization.acronym})` }}
+          </span>
+          <span class="h5 float-right text-secondary">
+            ({{ (getOrganizationTypeById(contactObj.organization.organizationTypeId) || {}).acronym }})
+          </span>
+        </div>
+      </div>
+      <div v-if="contactObj.organization && contactObj.organization.government" class="row">
+        <div class="col-12">
+          <div class="h4">
+            <b>{{ (getCountryByCode(contactObj.organization.government) || {}).name }}</b>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-6">
+          <Address :address="contactObj" />
+        </div>
+        <div class="col-6">
+          <div>
+            <a
+              v-for="email in contactObj.emails"
+              :key="email"
+              :href="`mailto:${email}`"
+            >{{ email }};
+            </a>
+          </div>
+          <div>
+            <a
+              v-for="phone in contactObj.phones"
+              :key="phone"
+              :href="`tel:${phone}`"
```

Comment:
you shoule encodeUriComponent on phone number. you have no ide what user provided.

```suggestion
              :href="`tel:${encodeURIComponent(phone)}`"
```